cmd/zpool/zpool_main.c

```diff
@@ -8510,6 +8542,13 @@ zpool_do_scrub(int argc, char **argv)
 		}
 	}
 
+	if ((cb.cb_date_start != 0 || cb.cb_date_end != 0) &&
```

Comment:
Probably should add an additiona check that cb_date_start is earlier than cb_date_end.

Reply:
Good idea. Added.